config/prod-root-config.yaml

```diff
@@ -0,0 +1,113 @@
+global:
+  TRAINED_ON: "root"    #used to create name of the specific version of a2rchi we're using
+  DATA_PATH: "/root/data/"
+  ACCOUNTS_PATH: "/root/.accounts/"
+  LOCAL_VSTORE_PATH: "/root/data/vstore/"
+  ACCEPTED_FILES: 
+    -".txt"
+    -".html"
+    -".pdf"
+
+interfaces:
+  chat_app:
+    PORT: 7861
+    EXTERNAL_PORT: 7684
+    HOST: "0.0.0.0" # either "0.0.0.0" (for public) or "127.0.0.1" (for internal)
+    HOSTNAME: "submit06.mit.edu"  # careful, this is used for the chat service
+    template_folder: "/root/A2rchi/A2rchi/interfaces/chat_app/templates"
+    static_folder: "/root/A2rchi/A2rchi/interfaces/chat_app/static"
+    num_responses_until_feedback: 3 #the number of responses given by A2rchi until she asks for feedback.
+  uploader_app:
+    PORT: 5001
+    HOST: "0.0.0.0" # either "0.0.0.0" (for public) or "127.0.0.1" (for internal)
+    template_folder: "/root/A2rchi/A2rchi/interfaces/uploader_app/templates"
+
+chains:
+  input_lists:
+    - empty.list
+    - miscellanea.list
+    - root-docs.list
+    # - root-forum.list
```

Comment:
Just leaving a comment / check in case you wanted to uncomment this for release? Just pointing it out in case it was commented out temporarily or something

Reply:
Meant to be commented. I would like to look more into the HTML scraper before including forum files because despite great potential and some well working examples the behavior seems still a bit unstable depending on the HTML formatting, especially when using HF embeddings. I wanted to merge the basic working branch before we add new functionalities.